llvm/test/MC/AMDGPU/gfx12_asm_ds.s

```diff
@@ -1929,6 +1929,12 @@ ds_bvh_stack_push4_pop1_rtn_b32 v1, v0, v1, v[2:5]
 ds_bvh_stack_push4_pop1_rtn_b32 v1, v0, v1, v[2:5] offset:1
 // GFX12: encoding: [0x01,0x00,0x80,0xdb,0x00,0x01,0x02,0x01]
 
+ds_bvh_stack_rtn_b32 v1, v0, v1, v[2:5]
```

Comment:
Nit:should really be in gfx12_asm_ds**_alias**.s and you only need one example of the instruction.